apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/AccountController.java

```diff
@@ -664,4 +672,34 @@ public Response enable2FA(
         return response.bind(dto).build();
     }
 
+    @Path("/count")
```

Comment:
account-statistic or account-summary better fit, i guess.

Reply:
updated to = /accounts/statistic